examples/neural_networks/plot_mlp_nonlinear.py

```diff
@@ -0,0 +1,46 @@
+"""
+=================================
+Non-linear Multi-layer Perceptron
+=================================
+
+Performs binary classification using multi-layer perceptron
+with l-bfgs algorithm. The prediction target is an XOR of the
```

Comment:
this docstring is out-dated.
I'm not sure this example is very helpful, given that we have non-linear datasets in the "classifier comparison".